build/run_container.sh

```diff
@@ -49,19 +49,28 @@ run_build_container() {
       cmd=(/bin/bash)
   fi
 
+  # In case of `minikube`, kube config stores full path to a certificates,
+  # thus the simples way to get working minikube in the build container is
+  # to bind original path to minikube settings to the container.
+  local path_to_minikube_dir="${HOME}/.minikube"
+  local minikube_dir_binding="-v ${path_to_minikube_dir}:${path_to_minikube_dir}"
+  if [ ! -d "${path_to_minikube_dir}" ]; then
+      minikube_dir_binding=""
+  fi
+
   docker run                                                      \
       --platform ${PLATFORM}                                      \
       ${extra_params}                                             \
       --rm                                                        \
       --net host                                                  \
       -e GITHUB_TOKEN="${github_token}"                           \
+      ${minikube_dir_binding}                                     \
       -v "${HOME}/.kube:/root/.kube"                              \
       -v "${PWD}/.go/pkg:/go/pkg"                                 \
       -v "${PWD}/.go/cache:/go/.cache"                            \
       -v "${PWD}:/go/src/${PKG}"                                  \
       -v "${PWD}/bin/${ARCH}:/go/bin"                             \
       -v "${PWD}/.go/std/${ARCH}:/usr/local/go/pkg/linux_${ARCH}" \
-      -v "${HOME}/.docker:/root/.docker"                          \
```

Comment:
Is removing the docker socket here intentional if yes, what would happen if the command that is being run in build container creates another container? Where would that container be created.

Reply:
With this change, if the command that is being run in build container creates another container, a run will successful in comparing with current behavior.

Reply:
sorry I didn't  understand this `a run will successful in comparing with current behavior.`

Reply:
Currently if you'll try to run some command within container, which will try to run another container, you'll get `ERROR: Cannot connect to the Docker daemon at unix:///path_to_socket. Is the docker daemon running?` error, it has been fixed in second commit of this PR. Exactly because of binding `~/.docker` directory.